.github/workflows/auto-assign-reviewers.yml

```diff
@@ -18,7 +18,9 @@ jobs:
             - name: Checkout repository
               uses: actions/checkout@v4
               with:
-                  fetch-depth: 0
+                  # fetch enough commits to include the BASE_SHA, without fetching the full repo history.
+                  # note: it's possible a PR will have more than 1000 commits, but extremely unlikely.
+                  fetch-depth: 1000
```

Comment:
I went with the poor man's solution of just fetching the last 1000 commits. This should be sufficient to cover PRs of any size, while still being considerably faster than fetching the entire repo's history.